website/project/views/node.py

```diff
@@ -781,6 +784,7 @@ def _view_project(node, auth, primary=False):
             },
             'alternative_citations': [citation.to_json() for citation in node.alternative_citations],
             'has_draft_registrations': node.has_active_draft_registrations,
+            'contributors': contributor_ids
```

Comment:
Where is this being used? I looked around the code but wasn't able to figure out why this was added in.